include/grpc/grpc_security.h

```diff
@@ -48,6 +48,7 @@ extern "C" {
 
 #define GRPC_X509_CN_PROPERTY_NAME "x509_common_name"
 #define GRPC_X509_SAN_PROPERTY_NAME "x509_subject_alternative_name"
+#define GRPC_X509_BASE64_FINGERPRINT_PROPERTY_NAME "x509_base64_fingerprint"
```

Comment:
I don't think that we should expose this property which is very much specific for your use case. Instead, we should surface the whole peer PEM cert (and possibly the cert chain but we can do that later).